RELEASE.md

```diff
@@ -17,6 +17,7 @@
 
 ## Bug fixes and other changes
 * Fix bug where `micropkg` manifest section in `pyproject.toml` isn't recognised as allowed configuration.
+* Commas surrounded by an even number of `"` and `'` characters will no longer split the arguments to `kedro run` options which take a list of nodes as inputs (`--from-nodes` and `--to-nodes`).
```

Comment:
```suggestion
* Commas surrounded by an even number of `[` and `]` characters will no longer split the arguments to `kedro run` options which take a list of nodes as inputs (`--from-nodes` and `--to-nodes`).
```

Reply:
Thanks for the reminder! Fixed in [a74a105](https://github.com/kedro-org/kedro/pull/2139/commits/a74a1056abcd22ae9501a12e888ee561c018f1d8).